graylog2-server/src/test/java/org/graylog/plugins/views/search/rest/scriptingapi/request/RequestedFieldTest.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+package org.graylog.plugins.views.search.rest.scriptingapi.request;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class RequestedFieldTest {
```

Comment:
What happens if we:
1. Provide a string that starts with ".", i.e. ".streams.smth".
2. Provide a string with multiple dots, i.e. "streams.outputs.title"
?
Can the test verify and document expected behavior in those 2 scenarios?

Reply:
This cases are handled on another level, namely in the TabularResponseCreator implementations. I will add these tests there.